etc/postgresql/values.yaml

```diff
@@ -172,7 +172,7 @@ postgresqlDataDir: /bitnami/postgresql/data
 # Define custom environment variables to pass to the image here
 extraEnv:
   - name: POSTGRES_MULTIPLE_DATABASES
-    value: managementportal,restsourceauthorizer,appconfig
+    value: managementportal,restsourceauthorizer,appconfig,appserver
```

Comment:
We added to this when we started, however I’m not sure whether it takes care of change management. Could you please create a dedicated database instead? It’s fine to use the same password. See e.g. the upload connect database.